package.json

```diff
@@ -86,8 +86,10 @@
   "dependencies": {
     "@types/http-link-header": "^1.0.1",
     "@types/node": "^18.0.0",
-    "cross-fetch": "^3.0.6",
     "http-link-header": "^1.0.2",
     "relative-to-absolute-iri": "^1.0.5"
+  },
+  "engines": {
```

Comment:
I agree with the removal of cross-fetch, but I would not fix the node version like this, and instead document it in the changelog.
In my experience, specifying the node version like this causes too many issues.

Reply:
>  In my experience, specifying the node version like this causes too many issues.

What kind of issues? It should only produce an advisory warning on incompatible versions, and I would expect that in most cases compatible versions are used since all node versions <18.x are now no longer maintained.

"Unless the user has set the [engine-strict config](https://docs.npmjs.com/cli/v10/using-npm/config#engine-strict) flag, this field is advisory only and will only produce warnings when your package is installed as a dependency." - https://docs.npmjs.com/cli/v10/configuring-npm/package-json#engines

Reply:
Yarn v1 (which I use) interprets it strictly.